common/src/main/java/xyz/wagyourtail/jsmacros/client/api/classes/Draw3D.java

```diff
@@ -241,54 +251,210 @@ public Box addPoint(PositionCommon.Pos3D point, double radius, int color) {
+        );
+    }
+
+    /**
+     * @param x top left
+     * @param y
+     * @param z
+     *
+     * @since 1.6.5
+     * @return
+     */
+    public Draw2D addDraw2D(double x, double y, double z) {
+        return addDraw2D(x, y, z, 0, 0, 0, 1, 1, 200, false, false);
     }
 
+    /**
+     * @param x
+     * @param y
+     * @param z
+     * @param width
+     * @param height
+     *
+     * @since 1.6.5
+     * @return
+     */
+    public Draw2D addDraw2D(double x, double y, double z, double width, double height) {
+        return addDraw2D(x, y, z, 0, 0, 0, width, height, 200, false, false);
+    }
 
     /**
-     * register so it actually shows up
+     * @param x
+     * @param y
+     * @param z
+     * @param xRot
+     * @param yRot
+     * @param zRot
+     *
+     * @since 1.6.5
+     * @return
+     */
+    public Draw2D addDraw2D(double x, double y, double z, double xRot, double yRot, double zRot) {
+        return addDraw2D(x, y, z, xRot, yRot, zRot, 1, 1, 200, false, false);
+    }
+
+    /**
+     * @param x
+     * @param y
+     * @param z
+     * @param xRot
+     * @param yRot
+     * @param zRot
+     * @param width
+     * @param height
+     *
+     * @since 1.6.5
+     * @return
+     */
+    public Draw2D addDraw2D(double x, double y, double z, double xRot, double yRot, double zRot, double width, double height) {
+        return addDraw2D(x, y, z, xRot, yRot, zRot, width, height, 200, false, false);
+    }
+
+    /**
+     * @param x
+     * @param y
+     * @param z
+     * @param xRot
+     * @param yRot
+     * @param zRot
+     * @param width
+     * @param height
+     * @param minSubdivisions
+     *
+     * @since 1.6.5
+     * @return
+     */
+    public Draw2D addDraw2D(double x, double y, double z, double xRot, double yRot, double zRot, double width, double height, int minSubdivisions) {
+        return addDraw2D(x, y, z, xRot, yRot, zRot, width, height, minSubdivisions, false, false);
+    }
+
+    /**
+     * @param x
+     * @param y
+     * @param z
+     * @param xRot
+     * @param yRot
+     * @param zRot
+     * @param width
+     * @param height
+     * @param minSubdivisions
+     * @param renderBack
+     *
+     * @since 1.6.5
+     * @return
+     */
+    public Draw2D addDraw2D(double x, double y, double z, double xRot, double yRot, double zRot, double width, double height, int minSubdivisions, boolean renderBack) {
+        return addDraw2D(x, y, z, xRot, yRot, zRot, width, height, minSubdivisions, renderBack, false);
+    }
+
+    /**
+     * @param x top left
+     * @param y
+     * @param z
+     * @param xRot
+     * @param yRot
+     * @param zRot
+     * @param width
+     * @param height
+     * @param minSubdivisions
+     * @param renderBack
+     *
      * @since 1.6.5
+     * @return
+     */
+    public Draw2D addDraw2D(double x, double y, double z, double xRot, double yRot, double zRot, double width, double height, int minSubdivisions, boolean renderBack, boolean cull) {
```

Comment:
Multipart suggestion
```suggestion
    public Surface addDraw2D(double x, double y, double z, double xRot, double yRot, double zRot, double width, double height, int minSubdivisions, boolean renderBack, boolean cull) {
```